### ISSUE TITLE
Crash near the end of game load time.Description: Rendering overlay

### ISSUE THREAD

**Issue by Pi0nStEaM** [MehVahdJukaar/modelfix-multi#15]
Here is the the crash log.

[crash-2023-01-27_00.03.20-client.txt](https://github.com/MehVahdJukaar/modelfix-multi/files/10515754/crash-2023-01-27_00.03.20-client.txt)

**Comment by MehVahdJukaar** [comment]
Remove the mod and your game should still crash

**Comment by Pi0nStEaM** [comment]
Testing in progress.
Thank you for a quick response!

**Comment by Pi0nStEaM** [comment]
What in the world.  You were right.  I'm going to _attempt_ to fix those issues before resuming.

**Comment by Pi0nStEaM** [comment]
So, I downgraded another unrelated mod version, added a missing dependency, and so far no issues related to this mod.  I guess this was a false positive?

**Comment by MehVahdJukaar** [comment]
Yeah this mod had been getting so many false reports because of a class it happens to alter gets loaded early and if mod loading has already failed an error related to this is thrown. Just cascading errors pretty much
